client/src/components/PageLogin/PageLogin.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { useState } from 'react'
+import { doLogInSaga } from '../../redux/actions/actionLogIn'
+import { useDispatch } from 'react-redux';
+
+const PageLogin = () => {
+
+  const [form, setForm] = useState({
+    email: '',
+    password: ''
+  })
+
+  const dispatch = useDispatch()
+
+  const onChangeForm = (e) => {
+    setForm({ ...form, [e.target.name]: e.target.value })
+  }
+
+  const request = async () => {
```

Comment:
It's better to start handler names with `on`, e.g. `onLogin`. In general, naming should be consistent: either you call all handlers `onSomething` to just `something`.

Reply:
fixed